sdk/state/ingest.go

```diff
@@ -162,7 +162,8 @@ func (c *Channel) ingestTxMetaToUpdateBalances(resultMetaXDR string) error {
 					continue
 				}
 				ledgerEntryAddress = tl.AccountId.Address()
-				ledgerEntryBalance = int64(tl.Balance)
+				liabilities := tl.Liabilities()
+				ledgerEntryBalance = int64(tl.Balance - liabilities.Selling)
```

Comment:
❗ One thing I missed when writing the original issue is that we need to take into account liabilities for native balances too. You could do that in this PR, or also in a separate PR.

Reply:
good point, added here
https://github.com/stellar/experimental-payment-channels/pull/311/commits/89aff901aa1a0a91aa7aef5f043ea85c3a7ff9c2